### PR TITLE
Add setting for remind on domain registration every day

### LINKED ISSUE #266: Option to send more registration reminders
As we still have many paid but not registered auction domains we would like to intensify the number of reminders sent to auction winners. Currently we have an option to send one reminder configurable days before registration deadline.

Add additional option to send the reminder every day after the first reminder until the domain is registered or deadline is reached.

### PR DESCRIPTION
**Pull request by yulgolem**
Closes #266 by adding a setting option to send reminders every day until domain registered or deadline reached.

### CODE REVIEW

File: app/models/setting.rb [hunk 1]
```diff
@@ -83,4 +88,29 @@ def self.invoice_reminder_in_days
...
+
+  def self.remind_on_domain_registration_everyday
+    value = Setting.find_by(code: :remind_on_domain_registration_everyday).value
+
+    if value == 'true'
+      true
+    elsif value == 'false'
+      false
+    end
+  end
```
[the remark this answers is not in the document]
Comment (yulgolem) [reply app/models/setting.rb]: Yep, like ternary `value == 'true' ? true : false`.
I assume the return value must always be boolean, so `false` is a must somewhere here.

**Comment by yulgolem** [comment]
Blocked by #382 , too many methods in Setting class.

**Comment by vohmar** [comment]
getting internal error while trying to access invoices in admin and user views with this branch. Seeing these errors in browser console:

* application-df390770aaa9c06f7103.js:2 GET https://auction.infra.tld.ee/invoices 500 (Internal Server Error)
* application-df390770aaa9c06f7103.js:2 Refused to apply inline style because it violates the following Content Security Policy directive: "style-src 'self' https://www.gstatic.com https: 'nonce-SGr/AYPqhKlzgOnRc/69BA=='". Either the 'unsafe-inline' keyword, a hash ('sha256-s9lhLDWLSxQf1WvI8Z+Nt0HEbYV+c8/VPa2Cpt09ho4='), or a nonce ('nonce-...') is required to enable inline execution.

**Comment by vohmar** [comment]
we still have internal error issues with offers view for regular user and jobs view for admin user.

errors from the logs
offers:
`Jan 14 10:48:56 docker e8654385b5c4[23704]: [a0ed4117-a378-48a2-b5bd-0419b70ff98e] [10.255.85.1] ActionView::Template::Error (undefined method `vat_rate' for nil:NilClass):`

jobs:
`Jan 14 10:50:25 docker e8654385b5c4[23704]: [89dbe457-42bf-4571-a26a-a7c5d4cbc5f8] [10.255.85.1] ActionView::Template::Error (undefined method `retrieve' for nil:NilClass):`

**Comment by vohmar** [comment]
offers is fixed - data issue caused by manipulating data in the db to create test cases (missing billing profile for active offer and trying to create an invoice for it)

**Comment by yulgolem** [comment]
> jobs:
> `Jan 14 10:50:25 docker e8654385b5c4[23704]: [89dbe457-42bf-4571-a26a-a7c5d4cbc5f8] [10.255.85.1] ActionView::Template::Error (undefined method `retrieve' for nil:NilClass):`

To fix jobs we need to re-deploy and re-run `rake:data_migratons:emind_on_domain_registration_everyday_setting` . There was typo in this task, thanks to @karlerikounapuu for notice.

**Pull request merged**